modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -706,99 +707,76 @@ public Table createTable(String name, Consumer<TableChange> tableInitChange) {
      *
      * @param name Table name.
      * @param tableInitChange Table configuration.
-     * @param exceptionWhenExist If the value is {@code true}, an exception will be thrown when the table already exists, {@code
-     *         false} means the existing table will be returned.
      * @return A table instance.
      */
-    private CompletableFuture<Table> createTableAsync(
-            String name,
-            Consumer<TableChange> tableInitChange,
-            boolean exceptionWhenExist
-    ) {
+    private CompletableFuture<Table> createTableAsyncInternal(String name, Consumer<TableChange> tableInitChange) {
```

Comment:
Here and for other asyncInternal methods. Seems that we might use same javadoc as for `org.apache.ignite.table.manager.IgniteTables#createTableAsync` with proper "If a table with the same name already exists..."

Reply:
I do not consider that all internal method should have duplicate public API javadoc.

Reply:
_private_ access already hint us about internal nature of this method )

Reply:
Method with internal suffix appear because all public API required check a stopping lock.
The internal method has a same functionality as a public one, but avoid a stopping lock.